src/Microsoft.Identity.Web/Resource/RegisterValidAudience.cs

```diff
@@ -61,7 +61,12 @@ internal class RegisterValidAudience
             SecurityToken securityToken,
             TokenValidationParameters validationParameters)
         {
-            JwtSecurityToken token = securityToken as JwtSecurityToken;
+            JwtSecurityToken? token = securityToken as JwtSecurityToken;
+            if (token == null)
```

Comment:
This is the kind of bugs you discover when you turn this feature on. THe compiler does NULL analysis and points out areas of the code that are potentially unsafe.